### PR TITLE
opcache monitor based on curl-json collectd

### PR DESCRIPTION
**Pull request by xp-1000**
add monitor for opcache

### CODE REVIEW

File: pkg/monitors/nodetool/monitor.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package nodetool
```
Comment (keitwb) [inline pkg/monitors/nodetool/monitor.go:1]: I think this is extraneous?

Comment (xp-1000) [reply pkg/monitors/nodetool/monitor.go]: my bad, fixed

**Review by keitwb — APPROVED** [review]
Looks good, I think you have got the hang of this.  This is a highly specific thing to monitor but it seems legitimate.

**Comment by keitwb** [comment]
Don't worry about the check_links CI failure, that always happens when adding a new monitor.

**Pull request merged**